src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -318,29 +326,6 @@ protected virtual Task<bool> ShouldProvidePreselectedItemsAsync(CompletionContex
             return CreateItems(completionContext, symbolToContextMap.Keys, symbol => symbolToContextMap[symbol], missingSymbolsMap, totalProjects, preselect, telemetryCounter);
         }
 
-        private static ImmutableArray<DocumentId> GetRelatedDocumentIds(Document document, int position)
```

Comment:
why this logic is no longer needed?

Reply:
There's a race around this SyntaxContext cache, between getting items and description, since the cached position is only checked before creating items. After I fixed that, there's nothing left in this so it was inlined.